test/network_policy_test.go

```diff
@@ -115,11 +608,132 @@ func doTestNetworkPolicy(
 
 			err = validateNetworkPolicy(ctx, aeroCluster)
 			Expect(err).ToNot(HaveOccurred())
-
-			err = deleteCluster(k8sClient, ctx, aeroCluster)
-			Expect(err).ToNot(HaveOccurred())
 		},
 	)
+
+	Context("customInterface", func() {
+		clusterNamespacedName := getClusterNamespacedName(
+			"np-custom-interface", multiClusterNs1,
```

Comment:
Let's use a different namespace for cluster and networkName.

Reply:
Test-case with different namespace seems not useful as we are mocking the CNI behaviour. 
Added a comment around the limitations of current test-suite for customInterfaces